### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Tuxemon/Tuxemon](https://hosted.weblate.org/projects/tuxemon/tuxemon/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 1]
```diff
@@ -259,7 +260,7 @@ msgid "generic_failure"
 msgstr "¡Falló!"
 
 msgid "attempting_capture"
-msgstr "Intentando capturar…"
+msgstr "Intentando capturar..."
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:263]: Why changing the Unicode character by three dots? Weblate seems to prefer the former. I think that at least we should discuss it and reach a consensus between all translation groups.

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 2]
```diff
@@ -1102,7 +1103,7 @@ msgstr "Zunna"
 
 # Monster categories
 msgid "anteater"
-msgstr "Oso hormiguero"
+msgstr "Amenaza"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: What?

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 3]
```diff
@@ -1613,11 +1635,11 @@ msgstr "No me queda mucho tiempo. Omnichannel está viniendo...\n"
...
 
 msgid "quicklyknight2"
-msgstr "Dos personas controlan Omnichannel. La junta de dirección es un chiste.\n"
-"Solo sé uno de los nombres, no obstante."
+msgstr ""
+"Dos personas controlan Omnichannel. La junta de dirección es un chiste.\n"
 
 msgid "quicklyknight3"
-msgstr "Allie. El nombre es Allie."
+msgstr "Allie. Allie es el que no quiere que sepas algo."
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: I think Allie is female. Also "sepas algo" is too ambiguous. It does not look realistic.
```suggestion
msgstr "Allie. Allie es la que no quiere que lo sepas."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 4]
```diff
@@ -1687,7 +1713,9 @@ msgid "cometomepls"
 msgstr "AHORA, TODO EL MUNDO, POR FAVOR VENID AL CENTRO DEL PUEBLO. ¡OMNICHANNEL QUIERE DIRIGIRSE A VOSOTROS!"
 
 msgid "hellolovelypeople2"
-msgstr "Sé que probablemente os preguntáis por qué mis Omni caballeros y yo estamos aquí.\n"
+msgstr ""
+"Sé que probablemente os preguntáis por qué mis Caballeros Omni y yo estamos "
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: I think "Omni caballeros" sounds better. Maybe it is better with an hyphen, or joined as one word.

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 5]
```diff
@@ -1789,7 +1820,8 @@ msgstr "Cariño, espera. Acabo de encender la televisión y echaban una rueda de
 "Deberíamos verla."
 
 msgid "ipausedit"
-msgstr "La he puesto en pausa para poder coger un vaso de agua y verla juntos."
+msgstr ""
+"He puesto la TV en pausa, para poder coger un vaso de agua y verla juntos."
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: I think either the previous translation or the following one are more idiomatic (nobody says TV aloud in Spanish):
```suggestion
"He pausado la tele para poder coger un vaso de agua y verla juntos."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 6]
```diff
@@ -1849,7 +1881,7 @@ msgid "okaythen"
 msgstr "¡De acuerdo! ¡Solo dame un segundo y curaré tus Tuxemon!"
 
 msgid "okaythen2"
-msgstr "Bien, ¡parece que tus Tuxemon están llenos de energía! ¡Aquí están!"
+msgstr "Bien, ¡parece que tus Tuxemon están mejor ahora ¡Aquí están!"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "Bien, ¡parece que tus Tuxemon están mejor ahora! ¡Aquí están!"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 7]
```diff
@@ -1950,10 +1982,15 @@ msgstr "<recovery_module>: GREETINGS ${{name}}. PLEASE UNDERSTAND THIS AREA NEVE
...
-"Blank: No. Lo siento: he cometido un error. Por favor, nunca hables de esto. Continúa, disfruta tu viaje aquí."
+msgstr ""
+"Blank: Mmmm. Hola. Lo siento: normalmente somos precavidos con los extraños. "
+"Solo recibimos visitas... a veces.\n"
+"Blank: Pero ya que estás aquí, quizás puedas hacerme un favor. Puede sonar "
+"extraño viniendo de mí, pero necesito saberlo...\n"
+"Blank: ¿Sabes quién soy? ¿Me has... visto alguna vez? Tengo curiosidad: mi "
+"nombre, ¿tengo un...\n"
+"Blank: No. Lo siento: he cometido un error. Por favor, no le diga a ninguno "
+"que le pregunté. Continúa, disfruta tu viaje aquí."
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: I think the older translation, although not literally the same as the original, is more idiomatic. In either case, "usted" is rarely used in peninsular Spanish, especially with young people:

```suggestion
"Blank: No. Lo siento: he cometido un error. Por favor, no le digas a nadie "
"que te he preguntado eso. Continúa, disfruta tu viaje aquí."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 8]
```diff
@@ -2455,15 +2493,15 @@ msgstr ""
 "Tienda de Arte!\n"
 "Desde que nos cortaron el presupuesto hemos tenido que vender las obras de "
 "arte en lugar de solo mostrarlas.\n"
-"La entrada cuesta 50${{currency}}. Va a los dividendos anuales de la compañía."
+"La entrada cuesta 50$. Va a los dividendos anuales de la compañía."
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: You should use the old translation. `${{currency}}` will be replaced with the proper money symbol of the campaign, with may be different from the dollar sign.

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 9]
```diff
@@ -2455,15 +2493,15 @@ msgstr ""
...
 
 msgid "spyder_cottonart_intro"
 msgstr ""
 "Bienvenido a la Galería de Arte de Pueblo Algodón. Ah... ¡quiero decir, a la "
 "Tienda de Arte!\n"
-"Desde que nos cortaron el presupuesto hemos tenido que vender las obras de "
+" Desde que nos cortaron el presupuesto hemos tenido que vender las obras de "
 "arte en lugar de solo mostrarlas.\n"
-"La entrada es 50${{currency}}. Va a los dividendos anuales de la compañía."
+" La entrada es 50$. Va a los dividendos anuales de la compañía."
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Same as above.

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 10]
```diff
@@ -2565,7 +2603,7 @@ msgstr ""
 msgid "hmmm2"
 msgstr ""
 "He oído que están haciendo un nuevo parque. Para \"crear espacios verdes en "
-"la zona\"."
+"la zona\""
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Why did you remove the period?

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 11]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
 
 msgid "spyder_flower_sandy_notweesher"
 msgstr "Oh, no importa. Veo que no tienes un Tweesher."
+
+msgid "spyder_route2_roddick1"
+msgstr "Cuando fui emboscado en la duna por SPIGHTER estuve muy asustado."
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: The passive here is slightly strange, but passable.

```suggestion
msgstr "Cuando fui emboscado en la duna por un SPIGHTER estuve muy asustado."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 12]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "spyder_route2_graf2"
+msgstr "Me sirvieron."
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: That is plain wrong. This sentence is a pun on the previous one, with talks about tennis. The pun should be preserved somehow, not translated literally.

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 13]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "spyder_wayfarer1_bismuth2"
+msgstr "¡Gorrón!"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: I am not sure about the context of this one. Is this translation appropriate?

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 14]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "spyder_route2_roddick2"
+msgstr "Oye, ¡no parecías arrastrado en absoluto!"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: "creeped" is not "arrastrado" but "asustado". It makes more sense if you look at the previous sentence in the dialog.
```suggestion
msgstr "Oye, ¡no parecías asustado en absoluto!"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 15]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "menu_down_key"
+msgstr "Tecla Bajar"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Nobody says that in Spanish. It could be "Flecha Abajo" or better, simply "Abajo" (which even makes sense for a controller).
```suggestion
msgstr "Abajo"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 16]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "menu_secondary_select_key"
+msgstr "Tecla Cancelar"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "Botón Cancelar"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 17]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "spyder_route4_wulf2"
+msgstr "Ooh, ¡me siento tan castrado!"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: NOBODY says this in Spanish. You shouldn't translate literally.

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 18]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "menu_left_key"
+msgstr "Tecla Izquierda"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "Izquierda"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 19]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "dark_robo"
+msgstr "Robo Oscuro"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: What is the context? Is this a robot? Because "robo" is "theft" in Spanish, so maybe we should use a different word.

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 20]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "spyder_citypark_maniac"
+msgstr "Phew, ¡lo hemos cagado!"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Nobody says "phew" in Spanish.
```suggestion
msgstr "Puf, ¡la hemos cagado!"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 21]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "dodge_increased"
+msgstr "el {stat} de {user} ha incrementado."
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: "incrementar" is transitive. Also they may be masculine and feminine stats, so using "el" and "incrementarse" (which should is conjugated different for each gender) is not always appropriate.

```suggestion
msgstr "{stat} de {user} ha aumentado."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 22]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "menu_up_key"
+msgstr "Tecla Subir"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "Arriba"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 23]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "menu_right_key"
+msgstr "Tecla Derecha"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "Derecha"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 24]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "menu_primary_select_key"
+msgstr "Tecla Acción"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "Botón Acción"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 25]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "menu_back_key"
+msgstr "Tecla Volver"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Not very sure about this one...
```suggestion
msgstr "Botón Retorno"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 26]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "options_new_input_key0"
+msgstr "Pulse una tecla para fijar su tecla de entrada nueva"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Too literal. I don't like the keyboard assumption here, but it is also in the original sentence:

```suggestion
msgstr "Pulsa una tecla para establecerla como nuevo valor de la entrada."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 27]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "options_new_input_key1"
+msgstr "Reinicie su juego para que los cambios surtan en efecto."
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "Reinicia el juego para que los cambios surtan efecto."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 28]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "xeon"
+msgstr "Xeón"
+
+msgid "spyder_route4_wulf1"
+msgstr "Ah, tan solo recoger algunos suplementos medicinales"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Use the context. This is the first sentence that a trainer says!

```suggestion
msgstr "Ah, solo estaba recogiendo algunos suplementos medicinales."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 29]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "spyder_wayfarer1_bismuth1"
+msgstr ""
+"¡Espero que no esté relacionado a aquellos tuxemon enfermos en las granjas "
+"Scoop!"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
"¡Espero que no esté relacionado con aquellos Tuxemon enfermos en las granjas "
"Scoop!"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 30]
```diff
@@ -3438,3 +3476,74 @@ msgstr ""
...
+
+msgid "hp_restored"
+msgstr "el hp del {user} ha sido recuperada"
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: HP is hit points. In Spanish it would be "puntos de golpe" or simply "salud" (health). You also are mixing genders here.
```suggestion
msgstr "La salud de {user} ha sido restaurada."
```

**Review by vnmabus — CHANGES_REQUESTED** [review]
Thank you for your contribution @cyphra ! I have several comments and suggestions. If you want to further contribute to Tuxemon's translation to Spanish we can coordinate the translation in the future using the translation channel in Tuxemon's Discord.

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 2 above)
Comment (cyphra) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Several translation avaliable

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 4 above)
Comment (cyphra) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Usually the translation change the order

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 5 above)
Comment (cyphra) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Yep. I prefer "la tele" 👍🏽

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 2 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: I don't see any other translation. Plus, from the context it is clear that it is "oso hormiguero", as it is the category of Aardart (https://www.fakemon.net/fakemon/fakemon1595937788DGN.png).

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 4 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: Translation is not a mechanical thing. When we have the choice we have to pick the option that sounds better in the target language. "Omni" in Spanish is always a prefix, so the best sounding words would be "Omnicaballeros", "Omni-Caballeros" or "Omni Caballeros". I think I prefer the first one: "Omnicaballeros". What do you think?

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 2 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "Oso hormiguero"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po [hunk 31]
```diff
@@ -1634,13 +1656,16 @@ msgstr "Sí, señora."
...
+"♪ Watch me nae nae ♪\n"
+"Esta canción es muy buena, ¿verdad?"
 
 msgid "nopenotyet"
 msgstr "Lo siento, alguien ha resultado herido en la ruta. Ahora mismo está cerrada."
 
 msgid "bothofyou"
-msgstr "Vosotros dos, venid fuera. Un contingente de Omnichannel requiere vuestra presencia."
+msgstr ""
+"Vosotros dos, venid fuera. Una representación de Omnichannel desea dirigirse "
```
Comment (vnmabus) [inline mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
"Vosotros dos, venid fuera. Un representante de Omnichannel desea dirigirse "
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 4 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
"Sé que probablemente os preguntáis por qué mis Omnicaballeros y yo estamos "
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 8 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
"La entrada cuesta 50${{currency}}. Va a los dividendos anuales de la compañía."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 9 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
" La entrada es 50${{currency}}. Va a los dividendos anuales de la compañía."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 10 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
"la zona\"."
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 12 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]: I don't know how to preserve the same meaning and the relation with the previous sentence. The best thing I can come with is
```suggestion
msgstr "El tenis se me da mejor..."
```
that means "I am better at tennis...". @cyphra, @Sanglorian, what do you think?

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 13 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "¡Qué mala suerte!"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 1 above)
Comment (bitcraft) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po:263]: lets stick to three dots for now

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 19 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "Robot Oscuro"
```

File: mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po (hunk 17 above)
Comment (vnmabus) [reply mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po]:
```suggestion
msgstr "Ooh, ¡me siento tan débil!"
```

**Comment by vnmabus** [comment]
I think you can apply the last suggestion and merge it.

**Review by bitcraft — APPROVED** [review]
